code/modules/reagents/chemistry/recipes/medicine.dm

```diff
@@ -1,4 +1,130 @@
 
+/******BRUTE******/
+
+/datum/chemical_reaction/trophazole
+	name = "Trophazole"
+	id  = /datum/reagent/medicine/trophazole
+	results = list(/datum/reagent/medicine/trophazole = 3)
+	required_reagents = list(/datum/reagent/copper = 1, /datum/reagent/acetone = 2,  /datum/reagent/phosphorus = 1, /datum/reagent/consumable/enzyme = 1)
+
+/datum/chemical_reaction/sal_acid
+	name = "Salicyclic Acid"
+	id = /datum/reagent/medicine/sal_acid
+	results = list(/datum/reagent/medicine/sal_acid = 1)
+	required_reagents = list(/datum/reagent/medicine/omnizine = 1, /datum/reagent/medicine/C2/libital = 1)
```

Comment:
Oh boy, time to force the CMO to hand over his hypospray at the start of every round if he doesn't want his chemists to lynch him and call him a terrible CMO (it's not like the CMO will actually USE his hypospray for anything; you're making the CMO's hypospray's default chem loadout be about as effective as injecting someone with 5u of water)!